gui/data_loc_info.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, parent: "ControlMain"):
         self.hBoxDPathParams1 = QtWidgets.QHBoxLayout()
         self.basePathLabel = QtWidgets.QLabel("Base Path:")
         self.base_path_ledit = QtWidgets.QLabel()  # leave editable for now
```

Comment:
This is editable but does nothing?

Reply:
the comment is misleading as QLabel() is not editable. fixed in https://github.com/NSLS-II/lsdc/pull/299/commits/d785be05470ca0d74e3bc46c144c7d40952727b7